static/app/components/events/interfaces/spans/utils.tsx

```diff
@@ -807,77 +803,6 @@ export function getSpanGroupBounds(
   }
 }
 
-export class SpansInViewMap {
-  spanDepthsInView: Map<string, number>;
-  treeDepthSum: number;
-  length: number;
-  isRootSpanInView: boolean;
-
-  constructor(isRootSpanInView: boolean) {
-    this.spanDepthsInView = new Map();
-    this.treeDepthSum = 0;
-    this.length = 0;
-    this.isRootSpanInView = isRootSpanInView;
-  }
-
-  /**
-   *
-   * @param spanId
-   * @param treeDepth
-   * @returns false if the span is already stored, true otherwise
-   */
-  addSpan(spanId: string, treeDepth: number): boolean {
-    if (this.spanDepthsInView.has(spanId)) {
-      return false;
-    }
-
-    this.spanDepthsInView.set(spanId, treeDepth);
-    this.length += 1;
-    this.treeDepthSum += treeDepth;
-
-    if (treeDepth === 0) {
-      this.isRootSpanInView = true;
-    }
-
-    return true;
-  }
-
-  /**
-   *
-   * @param spanId
-   * @returns false if the span does not exist within the span, true otherwise
-   */
-  removeSpan(spanId: string): boolean {
-    if (!this.spanDepthsInView.has(spanId)) {
-      return false;
-    }
-
-    const treeDepth = this.spanDepthsInView.get(spanId);
-    this.spanDepthsInView.delete(spanId);
-    this.length -= 1;
-    this.treeDepthSum -= treeDepth!;
-
-    if (treeDepth === 0) {
-      this.isRootSpanInView = false;
-    }
-
-    return true;
-  }
-
-  has(spanId: string) {
-    return this.spanDepthsInView.has(spanId);
-  }
-
-  getScrollVal() {
-    if (this.isRootSpanInView) {
-      return 0;
-    }
-
-    const avgDepth = Math.round(this.treeDepthSum / this.length);
-    return avgDepth * (TOGGLE_BORDER_BOX / 2) - TOGGLE_BUTTON_MAX_WIDTH / 2;
-  }
-}
-
```

Comment:
Deleted this since we now track all spans that are in view within `SpanTree` so this is obsolete